tests/e2e/test_hybrid_e2e.py

```diff
@@ -176,3 +176,25 @@ def test_hybrid_retriever_return_properties(driver: Driver) -> None:
     assert len(results.items) == 5
     for result in results.items:
         assert isinstance(result, RetrieverResultItem)
+
+
+@pytest.mark.usefixtures("setup_neo4j_for_retrieval")
+def test_hybrid_retriever_search_text_linear_ranker(
```

Comment:
I don't see the linear ranker being used in this test?

Reply:
You're right https://github.com/neo4j/neo4j-graphrag-python/pull/284/commits/b47b6b31df1ce01e01ce3d774f838180d2c91b7f